ansible/pbTestScripts/testScript.sh

```diff
@@ -13,21 +17,34 @@ processArgs()
 
 setupFiles()
 {
-	cd ~
+	cd $HOME
 	mkdir -p adoptopenjdkPBTests || true
 	cd adoptopenjdkPBTests
 	mkdir -p logFiles || true
 }
 
-# Takes in git URL as arg 1, foldername as arg 2
+# Takes in git URL as arg 1, foldername as arg 2, branchName as arg 3
 setupGit()
 {
-	cd ~/adoptopenjdkPBTests
-	if [ ! -d "$2" ]; then		#if folder doesn't exist
-   		git clone $1
-	else				#if it does, ensure up to date
-		cd "$2"
-    		git pull $1
+	cd $HOME/adoptopenjdkPBTests
+	if [ "$3" = "NULL" ]; then
```

Comment:
As a general comment, refactor the script so that $1, $2 and $3 are immediately assigned to meaningfully name variables.

Also, I'm not sure a single `=` sign is the safe way to check equality in shell